reading/comparison_operators.livemd

```diff
@@ -42,6 +42,7 @@ For example, if we check that `5` equals `5`, we return `true`.
 ```
 
 Be aware that the font of this lesson may shorten some operators.
+For example, the font may combine a `<` character and `=` character to appear as one `<=` character when it is actually two separate characters.
```

Comment:
Hopefully Livebook will remove ligatures, they have mentioned they are open to it so I should make a PR then we don't have to mention this.